l10n_it_fatturapa/models/account.py

```diff
@@ -124,32 +242,167 @@ def create(self, cr, uid, vals, context=None):
             create(cr, uid, vals, context)
 
 
-class account_payment_term(orm.Model):
-    _inherit = 'account.payment.term'
+class faturapa_activity_progress(orm.Model):
+    #_position = ['2.1.7']
+    _name = "faturapa.activity.progress"
 
     _columns = {
-        'fatturapa_pt_id': fields.many2one(
-            'fatturapa.payment_term', string="FatturaPA Payment Term"),
-        'fatturapa_pm_id': fields.many2one(
-            'fatturapa.payment_method', string="FatturaPA Payment Method"),
+        'fatturapa_activity_progress': fields.integer('Activity Progress'),
+        'invoice_id': fields.many2one(
+            'account.invoice', 'Related Invoice',
+            ondelete='cascade', select=True)
+    }
+
+
+class fattura_attachments(orm.Model):
+    _name = "fatturapa.attachments"
+    _description = "FatturaPA attachments"
+    _inherit = 'ir.attachment'
```

Comment:
Per non duplicare le tabelle e mantenere i dati principali degli allegati sempre in ```ir_attachment```, qui userei la stessa ereditarietà già usata in fatturapa_in e fatturapa_out:
```_inherits = {'ir.attachment': 'ir_attachment_id'}```